day_6/exercises/burrito.rb

```diff
@@ -1,19 +1,55 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping=(toppings)
+    @toppings = @toppings << toppings
```

Comment:
If you're only adding a single topping, what would be a better name for your parameter here?